apps/nextjs-website/src/app/solutions/[solutionSlug]/[...solutionSubPathSlugs]/page.tsx

```diff
@@ -74,9 +73,9 @@ const Page = async ({ params }: { params: Params }) => {
       isPageIndex: page.isIndex,
       pagePath: page.path,
       assetsPrefix: source.assetsPrefix,
-      gitBookPagesWithTitle,
-      spaceToPrefix: spaceToPrefixMap,
       urlReplaces: urlReplaceMap,
+      gitBookPagesWithTitle: [],
+      spaceToPrefix: [],
```

Comment:
maybe we should add also here all the internal links  like in detail guide page?